### PR TITLE
Feat/privacy score integration

### PR DESCRIPTION
**Pull request by jreluiz**
PR regarding privacy rules

### CODE REVIEW

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/DatabaseManager.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public static Connection getConnectionInstance() {
         try {
             if ((connection == null) || connection.isClosed()) {
                 connection = DriverManager
-                        .getConnection("jdbc:mysql://mysql-0.mysql.default.svc.cluster.local:3306/knowledge?"
+                        .getConnection("jdbc:mysql://10.98.170.151:3306/knowledge?"
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/DatabaseManager.java]: Sei que já estava assim antes, mas não é melhor colocarmos num arquivo de configuração?

Comment (jreluiz) [reply common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/DatabaseManager.java]: Ok, pode colocar em um arquivo de configuração.

Comment (nmsa) [reply common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/DatabaseManager.java]: Isto também é na linha do que disse aqui: 
https://github.com/eubr-atmosphere/tma-framework-p/pull/9/files#r320778349

Também seria bom manter a conversa em Inglês, uma vez que isto fica público.

/cc @brenofranca83 @joseadp @mpvieira

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/TrustworthinessScore.java [hunk 2]
```diff
@@ -1,17 +1,19 @@
 package eubr.atmosphere.tma.utils;
 
-public class TrustworthinessScore implements Score {
+public class TrustworthinessScore extends AbstractTrustworthinessScore implements Score {
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/TrustworthinessScore.java]: @jreluiz Aren't we removing this interface?
(cc @nmsa @brenofranca83 )

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/MessageExecute.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.List;
+
+import eubr.atmosphere.tma.data.Configuration;
+
+/**
+ * Message format to be sent to execute component 
+ * 
+ * @author JorgeLuiz
+ */
+public class MessageExecute {
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/MessageExecute.java]: @jreluiz Why do we need this class? It seems it is duplicated with what we already have on `AdaptationManager` class.
The idea is to plan to send only the planID on kafka queue, and then execute component searches the complete plan from the database. 

(cc  @brenofranca83 @nmsa )

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/TrustworthinessScore.java (hunk 2 above)
Comment (jreluiz) [reply common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/TrustworthinessScore.java]: > @jreluiz Aren't we removing this interface?
> (cc @nmsa @brenofranca83 )

@joseadp, Interface removed.

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/MessageExecute.java (hunk 3 above)
Comment (jreluiz) [reply common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/MessageExecute.java]: > @jreluiz Why do we need this class? It seems it is duplicated with what we already have on `AdaptationManager` class.
> The idea is to plan to send only the planID on kafka queue, and then execute component searches the complete plan from the database.
> 
> (cc @brenofranca83 @nmsa )

@joseadp, `AdaptationManager` class removed. 
I made some changes according to your suggestions, please validate.

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/DatabaseManager.java [hunk 4]
```diff
@@ -29,7 +29,7 @@ public static Connection getConnectionInstance() {
         try {
             if ((connection == null) || connection.isClosed()) {
                 connection = DriverManager
-                        .getConnection("jdbc:mysql://10.98.170.151:3306/knowledge?"
+                        .getConnection("jdbc:mysql://10.98.170.151:3306/knowledgenew?"
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/DatabaseManager.java]: @jreluiz , I believe this change was not supposed to be committed, right?

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/MessageExecute.java (hunk 3 above)
Comment (joseadp) [reply common/tma-utils/src/main/java/eubr/atmosphere/tma/utils/MessageExecute.java]: @jreluiz , I believe you meant `MessageExecute`.
I agree with it. Thanks!

**Comment by joseadp** [comment]
@jreluiz , I made the change of the connection string directly. If you agree, we can merge this PR.
(cc @nmsa  @brenofranca83 )

**Comment by jreluiz** [comment]
> @jreluiz , I made the change of the connection string directly. If you agree, we can merge this PR.
> (cc @nmsa @brenofranca83 )

@joseadp, I agree.
Thanks a lot.

**Comment by joseadp** [comment]
@nmsa @brenofranca83 this PR can also be merged.
(cc @jreluiz )

**Pull request closed without merging**